test/files/column_operators_binops.py

```diff
@@ -27,11 +27,11 @@ class A:
 
 eq1: "ColumnElement[Boolean]" = A.id == A.id
 eq2: "ColumnElement[Boolean]" = A.id == 1
-eq3: "ColumnElement[Boolean]" = 1 == A.id
+# eq3: "ColumnElement[Boolean]" = 1 == A.id
```

Comment:
why is this commented out?

Reply:
can't support 1==A.id in 1.4 or v2. This uses `int.__eq__` that returns bool